starknet_py/proxy/proxy_check.py

```diff
@@ -11,6 +11,8 @@
 from starknet_py.net.models import Address
 
 
+# fmt: off
+# docs-proxy-check: start
 class ProxyCheck(ABC):
```

Comment:
Why is formatting disabled here? :hmmm:

Reply:
This class is showcased in documentation and after adding `# docs-proxy-check` comments above and below, the formatter would insert some 2 blank lines between the last line of the class and `# docs-proxy-check: end`

Reply:
I see, `but` I don't think it is the best solution. That section will not be changed too often but still...

Reply:
Best alternative is to leave it like this then:

<img width="730" alt="image" src="https://user-images.githubusercontent.com/65187002/212865778-c913e9e9-44c5-4a42-a9ee-15cb5040d3f8.png">

Reply:
Not perfect, but probably better than disabling `black`. @cptartur what do you think?

Reply:
Doesn't look that bad. Couldn't we work around this with inline disables with `# fmt: off`  and `#fmt: on` around the affected lines?